### DISCUSSION TITLE
Rendering of BT-3201: Identifier of a tender

### DISCUSSION THREAD

**Discussion by Wout3P** [OP-TED/eForms-SDK#381] [Q&A] [answered]
Hello all,

I'm generating a preview for a notice 29 in sdk1.5 via the render notice API.
The tender identifier label (BT-3201) is rendered even though the corresponding element (TenderReference/cbc:ID) is not inside the xml.
BT-3201 is not a mandatory field if I look to the fields.json. So I would expect it not being shown when it is not in the xml.
Is this a rendering issue and will this be fixed or is this related to other parts in the xml? :)

Thanks for helping,

**Comment by pdonohoe** [comment, marked as answer]
Dear @Wout3P , Thank you for pointing out this error, it is indeed a rendering issue. It will be fixed in a future release of the SDK, and the fix will then be applied to previous SDK versions.
